.github/ISSUE_TEMPLATE/USER_STORY.yaml

```diff
@@ -0,0 +1,56 @@
+name: (Carbon core team ONLY) User story
+description:
+  Write a user story to begin solving their needs.
+title: '[YOUR TITLE]: Brief description'
+body:
+  - type: markdown
+    attributes:
+      value: "Avoid any type of solutions in this user story."
+  - type: markdown
+    attributes:
+      value: "Consider the following when writing Acceptance criteria for this story:
+
+      - Each product backlog item or user story should have at least one Acceptance criteria.
+
+      - Acceptance criteria defines a deliverable that can be completed in a single sprint.
+
+      - Each Acceptance criterion is independently testable.
+
+      - Include functional as well as non-functional criteria – when relevant.
+
+      - Team members write Acceptance criteria and the Product Owner verifies it."
+  - type: textarea
```

Comment:
Would it make sense to break this into 3 different text areas? 

- `As a [user role]`
- `I need to`
- `so that I can`

Not sure if GitHub would format it nicely, but just an idea